### ISSUE TITLE
Allow to_csv to write empty dataframe

### ISSUE THREAD

**Issue by nabil-becker** [aws/aws-sdk-pandas#2028]
Similar to #1921, Id like the to_csv to write an empty dataframe

We use the SDK to generate dynamic CSV files.

There are datasets that are correctly empty, so we'd like to generate a csv file with no rows.

In #1921 it seems as though the enhancement was rejected because they didn't want to add more arguments to the method.

Maybe we can add a property to the class to allow it to work with EmptyDataFrames?

```python
from awswrangler import s3

class GlueJob:

    def __init__(self, args: dict):
        s3.AllowEmptyDataFrame = True
```

We attempted as a workaround to add an empty row to dataframe, but that produces an undesirable file.
```csv
Column1|Column2|Column3|Column4|Column5
|||| # <= Dont want this there
```

**Comment by malachi-constant** [comment]
Thanks @nabil-becker. 
I'll put together a PR and we'll review the enhancement then.

**Comment by jaidisido** [comment]
Fixed in #2045 and available in next release
